packages/makeswift/package.json

```diff
@@ -43,6 +43,7 @@
     "open": "^8.4.0"
   },
   "devDependencies": {
+    "@babel/types": "^7.23.0",
```

Comment:
I used `pnpm import` to copy the lockfile. I am not sure why I had to add this to calm down some type errors that came from using pnpm.

Reply:
Yarn makes it possible to depend on transitive dependencies that aren't in package.json so it might've just gone unnoticed. It's one of the benefits of pnpm that it doesn't allow you to depend on something like that implicitly.